api/src/main/resources/packages.xml

```diff
@@ -1,54 +1,6 @@
 <org.openmrs.module.emrapi.metadata.MetadataPackagesConfig>
     <packages>
 
-        <org.openmrs.module.emrapi.metadata.MetadataPackageConfig>
-            <filenameBase>Reference_Application_Location_Tags</filenameBase>
-            <groupUuid>5987fa0f-9526-461c-8d43-28cb2e14933e</groupUuid>
-            <version>1</version>
-            <importMode>MIRROR</importMode>
-        </org.openmrs.module.emrapi.metadata.MetadataPackageConfig>
-
-        <org.openmrs.module.emrapi.metadata.MetadataPackageConfig>
-            <filenameBase>Reference_Application_Person_Attribute_Types</filenameBase>
-            <groupUuid>d5fb6341-caca-4742-b230-c9d979911793</groupUuid>
-            <version>1</version>
-            <importMode>MIRROR</importMode>
-        </org.openmrs.module.emrapi.metadata.MetadataPackageConfig>
-
-        <org.openmrs.module.emrapi.metadata.MetadataPackageConfig>
-            <filenameBase>Reference_Application_Visit_Types</filenameBase>
-            <groupUuid>a745ec2b-032c-4815-b091-d0339667b919</groupUuid>
-            <version>1</version>
-            <importMode>MIRROR</importMode>
-        </org.openmrs.module.emrapi.metadata.MetadataPackageConfig>
-
-        <org.openmrs.module.emrapi.metadata.MetadataPackageConfig>
-            <filenameBase>Reference_Application_Encounter_Types</filenameBase>
-            <groupUuid>56b191a8-0f71-44e9-b0ea-651f9da955ce</groupUuid>
```

Comment:
@jdegraft and @dkayiwa , apologies for the confusion. If there were any earlier unused zips they should be removed. And if there were any references to those zips in packages.xml they should be removed. @jdegraft ,  I don't think there are any such references to zips and any such references in this packages.xml. Can you revert this file back.